backends/xboxone/README.md

```diff
@@ -0,0 +1,152 @@
+# Xbox One WebDriver Server
+
+A [WebDriver][] server for Xbox One devices, implementing the subset of the
+WebDriver protocol necessary for [Karma][].  Add Xbox One devices to your
+[Selenium grid][]!
+
+Part of the [Generic WebDriver Server][] family.
+
+
+## Prerequisites
+
+To use these tools, you will need:
+ - Xbox One device in Dev Mode
```

Comment:
Update this to mention supported models more explicitly, or add a section above called "supported models"